main.py

```diff
@@ -337,6 +342,50 @@ def isPaused(self):
 	def isRunning(self):
 		return self._p is not None # FIXME: Check subproc state
 
+class AppVeyor(object):
+	def __init__(self, repo, branch, user):
+		self.gjson = json.loads(requests.get("https://ci.appveyor.com/api/projects/" + user + "/" + repo + "/branch/" + branch).text)
+
+	def GetJobs(self):
+		return self.gjson["build"]["jobs"]
+
+	def GetRelease(self, id):
+		idjson = json.loads(requests.get("https://ci.appveyor.com/api/buildjobs/" + str(id) + "/artifacts").text)
```

Comment:
What does the `id` in `idjson` stand for? Why not `artifacts_json` or `release_json` ?

Reply:
for debugging (printing the json and look at it) not needed anymore and variable name is not fitting, i will adjust these same as the last one.